packages/admin-api/lib/index.js

```diff
@@ -33,20 +33,29 @@ export default function GhostAdminAPI(options) {
 
     const config = Object.assign({}, defaultConfig, options);
 
+    // new GhostAdminAPI({host: '...'}) is deprecated
+    if (config.host) {
+        // eslint-disable-next-line
+        console.warn('GhostAdminAPI\'s `host` parameter is deprecated, please use `url` instead');
```

Comment:
Don't think we have to deprecate anything as we have not released a stable version for `admin-api` yet, so the only users that have to adapt to new usage of this parameter are us :thinking: